Automattic/newspack-plugin
# fix(ras-defaults): updates for design review feedback

### All Submissions:






### Changes proposed in this Pull Request:



With https://github.com/Automattic/newspack-popups/pull/1107, provides updates to the RAS setup wizard from design feedback (thanks, @evilluendas!).

Closes `1204167513009827/1204351495016571`.

### How to test the changes in this Pull Request:

Prior to testing, make sure you're running https://github.com/Automattic/newspack-popups/pull/1107 and run `wp option delete newspack_reader_activation_enabled` to reset your wizard to a state where you can revisit the RAS prompts' setup wizard.

#### Previewing shows values from your current input, even if unsaved)

Previewing a prompt no longer automatically saves your input, so you can properly preview your changes without committing them.

1. Go to **Newspack > Engagement > Reader Activation**, expand "Reader Activation Campaign", and click in to view the prompt wizard
2. Expand some prompts and make some changes to all input fields. 
3. Without saving, click "Preview prompt". Confirm that the preview window appears immediately and that the prompt rendered in the preview shows your unsaved changes.
4. Refresh the page without saving and confirm that your changes were NOT saved.

#### Preview button icon color

1. Expand a prompt card and click "Update prompt settings" to save.
2. Confirm that while in the loading state, the color of the preview icon turns from blue to grey, like this:

<img width="262" alt="Screen Shot 2023-05-02 at 6 15 06 PM" src="https://user-images.githubusercontent.com/2230142/235811361-fe04ee1d-acc9-4c7e-91a3-475afe0c77e9.png">

### Other information:
